### PR TITLE
feat/room-qr-code

### PR DESCRIPTION
**Pull request by ngomesodent**

### CODE REVIEW

File: expo/app/(tabs)/rooms/[id]/index.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Button from "../../../../components/Button";
+
+export default function TabsRooms() {
```
Comment (MAXOUXAX) [inline expo/app/(tabs)/rooms/[id]/index.tsx:3]: This shouldn't be called TabsRoom but rather "RoomView" or something like that

**Review by MAXOUXAX — CHANGES_REQUESTED** [review]
Looking good for now!

**Review by drmolixcool — APPROVED** [review]

File: expo/app/(tabs)/rooms/[id]/index.tsx (hunk 1 above)
Comment (ngomesodent) [reply expo/app/(tabs)/rooms/[id]/index.tsx:3]: My bad forgot to rename it. Done :)

File: expo/app/(tabs)/rooms/[id]/qrcode.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import { StyleSheet } from "react-native";
+import QRCode from "react-native-qrcode-svg";
+
+import Alert from "../../../../components/Alert";
+import Button from "../../../../components/Button";
+import { View } from "../../../../components/Themed";
+import { supabase } from "../../../../lib/supabase";
+import { generatedInvitationLink } from "./invite";
+
+export default function InvitationModal() {
```
Comment (MAXOUXAX) [inline expo/app/(tabs)/rooms/[id]/qrcode.tsx:13]:
```suggestion
export default function QRCodeModal() {
```

**Review by MAXOUXAX — APPROVED** [review]
Just one component name to change, but otherwise LGTM!

**Pull request merged**